src/{% if ci == 'Travis' %}.travis.yml{% else %}.t2d.yml{% endif %}.jinja

```diff
@@ -28,6 +28,13 @@ addons:
       - {{ package|tojson }}
       {%- endfor %}
 
+{%- if ci_extra_env %}
+env:
+  {%- for key, value in ci_extra_env.items() %}
+  - {{ key }}={{ value|tojson }}
+  {%- endfor %}
+{%- endif %}
+
```

Comment:
I would not change the Travis template: this fading away and let's not take risk with it.

Reply:
Removed in latest commit